### PR TITLE
Get all received delegations

### PR DESCRIPTION
**Pull request by acn-dgopa**
Get all receive delegations for a reportee

## Description
Get all the delegations where the coveredbypartyid is the reportee.

## Related Issue(s)
- #122 

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: backend/src/Altinn.Authorizationadmin/Altinn.AuthorizationAdmin.Core/Models/ReceivedDelegation.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        /// <summary>
+        /// Gets or sets the name of the resource in a delegation
+        /// </summary>
+        [JsonPropertyName("offeredbypartyid")]
+        public int OfferedByPartyId { get; set; }
+
+        /// <summary>
+        /// Resources that were received
+        /// </summary>
+        public List<ServiceResource> Resources { get; set; }
```
Comment (jonkjetiloye) [inline backend/src/Altinn.Authorizationadmin/Altinn.AuthorizationAdmin.Core/Models/ReceivedDelegation.cs:26]: Listing the entire ServiceResource model might be a bit overkill for what the function is to be used for?
But organization number for the OfferedBy might be nice.

Comment (jonkjetiloye) [reply backend/src/Altinn.Authorizationadmin/Altinn.AuthorizationAdmin.Core/Models/ReceivedDelegation.cs:26]: Documentation summary for ReporteeName and OfferedByPartyId needs an update

File: backend/src/Altinn.Authorizationadmin/Altinn.AuthorizationAdmin.Core/Services/Implementation/DelegationsService.cs [hunk 2]
```diff
@@ -45,16 +45,88 @@ public async Task<List<OfferedDelegations>> GetAllOfferedDelegations(int offered
...
+                    }
+
+                    receivedDelegations.Add(receivedDelegation);
+                }
+            }
+
+            return receivedDelegations;
+        }
+
+        private async Task<List<ServiceResource>> GetResources(List<string> resourceIds)
```
Comment (jonkjetiloye) [inline backend/src/Altinn.Authorizationadmin/Altinn.AuthorizationAdmin.Core/Services/Implementation/DelegationsService.cs:116]: GetResources could be moved to a ResourceRegistryHelper/Wrapper class as we will need both this and other integrations with the resource registry across multiple services in access management

File: backend/src/Altinn.Authorizationadmin/Altinn.AuthorizationAdmin.Core/Models/ReceivedDelegation.cs (hunk 1 above)
Comment (acn-dgopa) [reply backend/src/Altinn.Authorizationadmin/Altinn.AuthorizationAdmin.Core/Models/ReceivedDelegation.cs:26]: ok, I was working on this now. Let me then push only the other changes

**Review by jonkjetiloye — APPROVED** [review]
Just the minor comments which have been reintroduced from the other PR

File: backend/src/Altinn.Authorizationadmin/Altinn.Authorizationadmin/Controllers/DelegationsController.cs [hunk 3]
```diff
@@ -82,7 +82,7 @@ public async Task<ActionResult> Post([FromBody] List<Rule> rules)
         /// <response code="400">Bad Request</response>
         /// <response code="500">Internal Server Error</response>
         [HttpPost]
-        [Authorize(Policy = AuthzConstants.ALTINNII_AUTHORIZATION)]
+        ////[Authorize(Policy = AuthzConstants.ALTINNII_AUTHORIZATION)]
```
[the remark this answers is not in the document]
Comment (acn-dgopa) [reply backend/src/Altinn.Authorizationadmin/Altinn.Authorizationadmin/Controllers/DelegationsController.cs]: looks like this is gone after merging with main

**Pull request merged**